gulpfile.js

```diff
@@ -62,7 +62,7 @@ gulp.task('templates', function() {
 
 gulp.task('copy', function() {
   gulp.src('./assets/json/*.json').pipe(gulp.dest('./dist'));
-  gulp.src('./node_modules/govuk_template_mustache/assets/images/*/**').pipe(gulp.dest('./dist/images'));
+  gulp.src('./node_modules/govuk_template_mustache/assets/images/*.**').pipe(gulp.dest('./dist/images'));
```

Comment:
Interesting you said that. For some reason, Gulp was pulling images from a different folder when I used **/*. I therefore changed it to *.** so it could capture just any file in that specific images folder (there are no subfolders there, all but one file are .png, the only other file being an .ico file.